test/test_unit.jl

```diff
@@ -579,6 +579,42 @@ isdir(outdir) && rm(outdir, recursive=true)
     end
   end
 
+  @timed_testset "Consistency check for Godunov flux" begin
+    # Set up equations and dummy conservative variables state
+    equation = InviscidBurgersEquation1D()
+    u = SVector(42)
```

Comment:
```suggestion
    u = SVector(42.0)
```
I would use a float value here, since this is our typical data type